llvm/lib/Support/VirtualFileSystem.cpp

```diff
@@ -2933,8 +2933,21 @@ recursive_directory_iterator::increment(std::error_code &EC) {
   return *this;
 }
 
+void TracingFileSystem::printImpl(raw_ostream &OS, PrintType Type,
```

Comment:
Should this print the current stat values?

Reply:
Good idea, implemented in the new commit.